.github/workflows/manual-deploy-testnet-faucet.yml

```diff
@@ -42,10 +42,10 @@ jobs:
         uses: docker/setup-buildx-action@v1
 
       - name: 'Sets env vars for testnet'
-        if: ${{ inputs.testnet_type == 'testnet' }}
+        if: ${{ inputs.testnet_type == 'uat-testnet' }}
         run: |
-          echo "FAUCET_BUILD_TAG=testnetobscuronet.azurecr.io/obscuronet/faucet_testnet:latest" >> $GITHUB_ENV
-          echo "TESTNET_ADDR=erpc.testnet.obscu.ro" >> $GITHUB_ENV
+          echo "FAUCET_BUILD_TAG=testnetobscuronet.azurecr.io/obscuronet/uat_faucet_testnet:latest" >> $GITHUB_ENV
```

Comment:
not faucet_uat_testnet?

Reply:
Looks like all other images are prepended at the start - should we change it ?

Reply:
Yeah changing it everywhere to normalize it